src/UI/HTMLTooltip.js

```diff
@@ -301,6 +311,9 @@ export class HTMLTooltip {
                 document.fonts.load("normal 13px 'DDG_ProximaNova'"),
                 document.fonts.load("bold 13px 'DDG_ProximaNova'")
             ]).then(() => {
+                const offscreen = {left: -1000, top: -1000}
```

Comment:
Have you considered applying these as default in the stylesheet? Feels cleaner/safer.